migrations/db/migrate.sh

```diff
@@ -56,3 +56,6 @@ fi
 
 # once done with everything, reset stats from init
 psql -v ON_ERROR_STOP=1 --no-password --no-psqlrc -U supabase_admin -c 'SELECT extensions.pg_stat_statements_reset(); SELECT pg_stat_reset();' || true
+
+# Create migration-output.sql file
+cd "$db"/migrations && ls -1 > /opt/migrations/db/migration-output.sql
```

Comment:
```suggestion
ls -1 "$db/migrations" > /opt/migrations/db/migration-output.sql
```

I think no need to `cd` if you pass the directory to `ls`